test/client/fixtures/sandbox/storages-test.js

```diff
@@ -0,0 +1,424 @@
+var StorageWrapper = hammerhead.get('./sandbox/storages/wrapper');
+var settings       = hammerhead.get('./settings');
+
+var storageSandbox = hammerhead.sandbox.storageSandbox;
+var iframeSandbox  = hammerhead.sandbox.iframe;
+var Promise        = hammerhead.Promise;
+var isIE           = hammerhead.utils.browser.isIE;
+
+var storageWrapperKey = 'hammerhead|storage-wrapper|' + settings.get().sessionId + '|example.com';
+
+QUnit.testStart(function () {
+    // NOTE: Clean up storage wrappers
+    window.localStorage.clear();
+    window.sessionStorage.clear();
+    storageSandbox.localStorage.clear();
+    storageSandbox.sessionStorage.clear();
+
+    iframeSandbox.on(iframeSandbox.IFRAME_READY_TO_INIT_EVENT, initIframeTestHandler);
+    iframeSandbox.off(iframeSandbox.IFRAME_READY_TO_INIT_EVENT, iframeSandbox.iframeReadyToInitHandler);
+});
+
+QUnit.testDone(function () {
+    iframeSandbox.off(iframeSandbox.IFRAME_READY_TO_INIT_EVENT, initIframeTestHandler);
+});
+
+function waitStorageUpdated () {
+    return new Promise(function (resolve) {
+        window.setTimeout(resolve, 100);
+    });
+}
+
+function waitIframeLoad (iframe) {

```

Comment:
I suppose the [waitForIframe](https://github.com/AlexanderMoskovkin/qunit-harness#wait-for-an-iframe-action) is the same